.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,46 @@
+name: run-tests
+
+on:
+    push:
+    pull_request:
+    schedule:
+        -   cron: '0 0 * * *'
+
+jobs:
+    run-tests:
+        runs-on: ubuntu-latest
+        strategy:
+            fail-fast: false
+            matrix:
+                php: [ 8.0, 7.4 ]
+                laravel: [ 8.*, 7.* ]
```

Comment:
The package only supports Laravel 8 (see composer.json), so running the tests on Laravel 7 can be removed from atrix